example/exporter/kube-state-metrics/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/scrape: 'true'
+  name: kube-state-metrics
+  labels:
+    app: kube-state-metrics
```

Comment:
kube-state-metrics is regarded as an add-on to kubernetes so it's fine to do `k8s-app: kube-state-metrics`